app/main/helpers/login_helpers.py

```diff
@@ -45,3 +50,7 @@ def get_user_dashboard_url(current_user_):
         return url_for('external.admin_dashboard')
 
     return None
+
+
+def dos6_live(params):
+    return datetime.utcnow().strftime('%Y-%m-%d') >= DOS6_GO_LIVE_DATE or params.get('show_dos6_live') == 'true'
```

Comment:
Do we need to convert `datetime.utcnow()` to a string? Could we compare it as a datetime?

Reply:
We could but my thinking was that it might be quicker to compare two strings rather than two date objects? I'm happy to do just do date comparison though as it is probably more appropriate.

Reply:
BTW - I forgot to mention it, but I really like the idea of having a query parameter to allow us to test this.

Reply:
I'm not sure I really needed to worry about performance as its a simple operation, oh well. I've changed it to date as that's what it should have been anyway.